src/Core/Content/Media/File/DownloadResponseGenerator.php

```diff
@@ -53,7 +52,7 @@ public function getResponse(
             $url = $fileSystem->temporaryUrl($path, (new \DateTime())->modify($expiration));
 
             return new RedirectResponse($url);
-        } catch (UnableToGenerateTemporaryUrl) {
+        } catch (\Throwable) {
```

Comment:
🤔 Can't we get any more precise? Before it was just catching a specific error, now it's like "If anything bad happens, just continue" without any message. Throwable without any kind of message doesn't feel right to me.

Reply:
If you've got a good idea, bring it on.

Of course, one could also argue that the composer package is messing up here.

The problem is that you don't know which Flysystem adapters will be configured later in the shop and which exceptions/throwables they will then throw.

Of course, you could do the following, but that would basically correspond to the current solution of the PR.
`... catch (UnableToGenerateTemporaryUrl|\Throwable) ...`

Reply:
Would it be possible to catch `FilesystemException` (or at least `Exception`) instead of `Throwable`?
Catching all Throwable instances also includes PHP Errors (e.g. TypeError, ParseError, memory issues), which usually indicate programming or system-level problems. Swallowing these would make debugging much harder and could hide real defects. Adding logging so we have enough context in case something goes wrong would also be good. This keeps the handling robust without suppressing critical errors :)